packages/contracts-bedrock/src/L2/interfaces/IOptimismSuperchainERC20Factory.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IOptimismERC20Factory } from "./IOptimismERC20Factory.sol";
```

Comment:
nit on import style, even tho eventually we should go back to relative paths so external projects can import this project without it breaking their remappings